app/views/rents_mailer/new_rent_notification.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Rents#new_rent_notification</h1>
+
+<p>
+  User: <%= @rent.user['first_name'] %>, <%= @rent.user['last_name'] %>
+<ul>
+<li>Book Title: <%= @rent.book['title'] %></li>
+<li>Book Author: <%= @rent.book['author'] %></li>
+<li>Book Genre: <%= @rent.book['genre'] %></li>
+<li>From: <%= @rent['from'] %></li>
+<li>To: <%= @rent['to'] %></li>
```

Comment:
same in here
You can read this for a little more info https://stackoverflow.com/questions/40765114/activerecord-accessor-dot-vs-bracket